Mathlib/CategoryTheory/Limits/Shapes/Types.lean

```diff
@@ -396,12 +399,12 @@ theorem productIso_inv_comp_π {J : Type v} (F : J → TypeMax.{v, u}) (j : J) :
   limit.isoLimitCone_inv_π (productLimitCone.{v, u} F) ⟨j⟩
 #align category_theory.limits.types.product_iso_inv_comp_π CategoryTheory.Limits.Types.productIso_inv_comp_π
 
-namespace UnivLE
+namespace Small
 
 /--
-A variant of `productLimitCone` using a `UnivLE` hypothesis rather than a function to `TypeMax`.
+A variant of `productLimitCone` using a `Small` hypothesis rather than a function to `TypeMax`.
 -/
-noncomputable def productLimitCone {J : Type v} (F : J → Type u) [UnivLE.{v, u}] :
+noncomputable def productLimitCone {J : Type v} (F : J → Type u) [Small.{u} J] :
     Limits.LimitCone (Discrete.functor F) where
   cone :=
     { pt := Shrink (∀ j, F j)
```

Comment:
It's more cumbersome than I expected to use Shrink J here. Changing the indexing type of a type family is a bit "evil" as you need to use Eq.rec to cast between two equal types, which happens here because equivShrink and equivShrink.symm aren't definitionally inverse to each other. This is a major difference with ULift.up/down, which are, so using two universes `u` and `v` with `UnivLE.{u,v}` will never be as convenient as using two universes `u` and `max u w`.